hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/push/DelayedEventBusPushStrategy.java

```diff
@@ -203,14 +161,79 @@ public void run() {
 
         }
 
-        private void doDispatch(final List<Event> events, final WrappedSession wrappedSession) {
+        private void addRolloutChangeEvents(final Optional<RolloutChangeEvent> rolloutChangeEvent,
+                final Optional<RolloutGroupChangeEvent> rolloutGroupChangeEvent, final List<TenantAwareEvent> events) {
+            rolloutChangeEvent.ifPresent(events::add);
+            rolloutGroupChangeEvent.ifPresent(events::add);
+
+        }
+
+        private Optional<RolloutChangeEvent> getRolloutChangeEvent() {
+            final Iterator<RolloutEventKey> rolloutIterator = rolloutEvents.iterator();
```

Comment:
@michahirsch this is wrong. We have to collect all events not only the first one. So we will lost rollout change events...